sbpy/data/tests/test_orbit_remote.py

```diff
@@ -112,12 +108,13 @@ def test_break(self):
             Orbit.from_mpc('does not exist')
 
 
-@pytest.mark.skipif('pyoorb is None')
 @pytest.mark.remote_data
 class TestOOTransform:
     def test_oo_transform(self):
         """ test oo_transform method"""
 
+        pytest.importorskip("pyoorb")
```

Comment:
You have importorskip before all the tests, do you still need one here?

Reply:
The only importorskip I see before all the tests is for astroquery, this one is just or pyoorb.